src/test/java/com/lpvs/controller/GitHubWebhooksControllerTest.java

```diff
@@ -148,7 +148,7 @@ public void wrongSecretTest() {
                         + "}"
                         + "}";
         try {
-            gitHubWebhooksController.setProps();
+            gitHubWebhooksController.initializeGitHubSecret();
```

Comment:
@o-kopysov Does this also relates to the Javadoc feature, or was it just some fix or refactoring that came to your attention?

Reply:
While preparing Javadocs, I did some name refactoring to make function names more understandable.